src/quo/core.cljs

```diff
@@ -197,6 +198,9 @@
 (def color-picker quo.components.colors.color-picker.view/view)
 (def color quo.components.colors.color.view/view)
 
+;;;; Common
+(def drawer-bar quo.components.common.drawer-bar.view/view)
```

Comment:
is this in the design system? 🤔

Reply:
Yes, it's under `iOS native components`. I attached the link to the design in the PR description.

Adding it here too [Figma Link](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=2500%3A41337&mode=design&t=xnQPYGXzw9ub7TVB-1) 👍

Reply:
perhaps we change the ns to quo.components.ios.drawer-bar.view/view or something like this to better match/ make it easier to find. We can also ask designers to update this change and align it then

Reply:
No worries. 

> perhaps we change the ns to quo.components.ios.drawer-bar.view/view or something like this to better match/ make it easier to find. We can also ask designers to update this change and align the codebase and figma file then

I placed it in `common` as this component is actually used in both Android and iOS but I agree with the ns `quo.components.ios.drawer-bar.*`. It's more aligned with Figma. I will update it 👍